code/modules/antagonists/nukeop/equipment/borgchameleon.dm

```diff
@@ -62,6 +62,8 @@
 			to_chat(user, "<span class='notice'>\the [src] is recharging.</span>")
 			return
 		animation_playing = TRUE
+		if(!fulp_borg_chameleon_menu(user)) //FULPSTATION SYNDICATE MEDBORG UPDATE by Surrealistik March 2020
```

Comment:
this should come BEFORE the if statement that says that the projector is still recharging, or else you can't ever turn your projector back on again (it'll always be "recharging") if it turns off (under certain conditions? idk, that was just what I found during testing)

Reply:
I shifted it above animation_playing